custom-video-player/script.js

```diff
@@ -4,6 +4,8 @@
 const progress = document.getElementById('progress');
 const timestamp = document.getElementById('timestamp');
 
+var a = foo
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://release.k8s.dev.codacy.org/gh/fjrdomingues/vanillawebprojects/pullRequest?prid=5787)
```suggestion
var a = foo;
```